internal/testing/e2e/e2e

```diff
@@ -17,13 +17,22 @@
 
 set -euf -o pipefail
 
-SCRIPT_DIR=$(cd $(dirname $0); pwd)
-GUAC_DIR=$(cd ${SCRIPT_DIR}/../../..; pwd)
+SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
+GUAC_DIR=$(cd "${SCRIPT_DIR}"/../../..; pwd)
+
+LOG_DIR="${GUAC_DIR}/internal/testing/e2e/logs"
+LOG_FILE="${LOG_DIR}/logfile.log"
+SCHEMA_BACKUP="${LOG_DIR}/schema_backup.sql"
+
+# Create the log directory if it doesn't exist
+mkdir -p "$LOG_DIR"
+
+echo "Script started" | tee -a "$LOG_FILE"
 
 guac_data_hash="254d737ebc42ca32f95033e93370f5427e3fce64"
 
-echo @@@@ Installing gql Python package
-pip install gql[all]
```

Comment:
keep this the same if its causing errors?

Reply:
Sorry, used this for testing, was cleaning it up as you were giving the comment. Thanks!